### PR TITLE
Various Paper Revisions

### PR DESCRIPTION
**Pull request by mandli**
Addresses some of the comments from the editor and the first reviewer.

### CODE REVIEW

File: papers/clawpack-5x/development.tex [hunk 1]
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
 implementation choices in the code.  Once the tests are passing and it
 is agreed that the code is acceptable, a maintainer merges it.
 
+\revised{An additional benefit of using the GitHub platform is that any version
+of the code is accessible either through the command line \texttt{git}
+interface, through the GitHub website, or a number of available applications 
+on all widely used platforms.  More importantly however is the ability to tag a
+particular version of a repository with a digital object identifier (DOI) via a
+GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
```
Comment (ketch) [inline papers/clawpack-5x/development.tex]: "via a Github and Zonodo" -> via Github and Zenodo

File: papers/clawpack-5x/development.tex [hunk 2]
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
 implementation choices in the code.  Once the tests are passing and it
 is agreed that the code is acceptable, a maintainer merges it.
 
+\revised{An additional benefit of using the GitHub platform is that any version
+of the code is accessible either through the command line \texttt{git}
+interface, through the GitHub website, or a number of available applications 
+on all widely used platforms.  More importantly however is the ability to tag a
```
Comment (ketch) [inline papers/clawpack-5x/development.tex]: "importantly" -> important

File: papers/clawpack-5x/development.tex [hunk 3]
```diff
@@ -193,6 +193,16 @@ \subsection{Contributing}
...
+particular version of a repository with a digital object identifier (DOI) via a
+GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
+version of software see \url{http://guides.github.com/activities/citable-code/}}.
+The combination of these abilities address the issues of availability of
+specific versions of the software and are citable in the scientific literature.}
```
Comment (ketch) [inline papers/clawpack-5x/development.tex]: Something is grammatically off with the final sentence, but I'm not completely sure what was intended.

Comment (mandli) [inline papers/clawpack-5x/development.tex]: I wanted to directly point out that these capabilities address the editor's question.  Revised to hopefully make this not awful.

File: papers/clawpack-5x/development.tex [hunk 4]
```diff
@@ -196,12 +196,14 @@ \subsection{Contributing}
 \revised{An additional benefit of using the GitHub platform is that any version
 of the code is accessible either through the command line \texttt{git}
 interface, through the GitHub website, or a number of available applications 
-on all widely used platforms.  More importantly however is the ability to tag a
-particular version of a repository with a digital object identifier (DOI) via a
+on all widely used platforms.  More important however is the ability to tag a
+particular version of a repository with a digital object identifier (DOI) via
 GitHub and Zonodo\footnote{For a guide on creating a DOI to a particular
```
Comment (ketch) [inline papers/clawpack-5x/development.tex]: Zonodo -> Zenodo

**Comment by ketch** [comment]
Looks good to me after the one tiny spelling correction.

**Comment by mandli** [comment]
Addresses a number of issues brought up by reviewers.

**Pull request merged**